pybossa/view/projects.py

```diff
@@ -400,7 +400,9 @@ def respond(errors):
         'data_classification': {
             'input_data': form.input_data_class.data,
             'output_data': form.output_data_class.data
-        }
+        },
+        "allow_taskrun_edit": False,
+        "allow_anonymous_contributors": False
```

Comment:
Is this setting anonymous contributors to always default to false? If so, does this affect the projects or worker access to them?

Reply:
no impact as anonymous contribution is not permitted in gigwork.